.travis.yml

```diff
@@ -0,0 +1,57 @@
+        - ARCH=armhf INCLUDE=wget QEMU_ARCH=arm SUITE=vivid UNAME_ARCH=armv7l VERSION=15.04
+        - ARCH=i386 INCLUDE=wget QEMU_ARCH=i386 SUITE=vivid UNAME_ARCH=i386 VERSION=15.04
+        - ARCH=powerpc INCLUDE=wget QEMU_ARCH=ppc SUITE=vivid UNAME_ARCH=powerpc VERSION=15.04
+        - ARCH=ppc64el INCLUDE=wget QEMU_ARCH=ppc64 SUITE=vivid UNAME_ARCH=ppc64el VERSION=15.04
+
+        - ARCH=amd64 INCLUDE=iproute2,wget QEMU_ARCH=x86_64 SUITE=wily UNAME_ARCH=x86_64 VERSION=15.10
+        - ARCH=arm64 INCLUDE=iproute2,wget QEMU_ARCH=aarch64 SUITE=wily UNAME_ARCH=arm64 VERSION=15.10
+        - ARCH=armhf INCLUDE=iproute2,wget QEMU_ARCH=arm SUITE=wily UNAME_ARCH=armv7l VERSION=15.10
+        - ARCH=i386 INCLUDE=iproute2,wget QEMU_ARCH=i386 SUITE=wily UNAME_ARCH=i386 VERSION=15.10
+        - ARCH=powerpc INCLUDE=iproute2,wget QEMU_ARCH=ppc SUITE=wily UNAME_ARCH=powerpc VERSION=15.10
+        - ARCH=ppc64el INCLUDE=iproute2,wget QEMU_ARCH=ppc64 SUITE=wily UNAME_ARCH=ppc64el VERSION=15.10
+
+        - ARCH=amd64 INCLUDE=iproute2,wget QEMU_ARCH=x86_64 SUITE=xenial UNAME_ARCH=x86_64 VERSION=16.04
+        - ARCH=arm64 INCLUDE=iproute2,wget QEMU_ARCH=aarch64 SUITE=xenial UNAME_ARCH=arm64 VERSION=16.04
+        - ARCH=armhf INCLUDE=iproute2,wget QEMU_ARCH=arm SUITE=xenial UNAME_ARCH=armv7l VERSION=16.04
+        - ARCH=i386 INCLUDE=iproute2,wget QEMU_ARCH=i386 SUITE=xenial UNAME_ARCH=i386 VERSION=16.04
+        # - ARCH=powerpc INCLUDE=iproute2,wget QEMU_ARCH=ppc SUITE=xenial UNAME_ARCH=powerpc VERSION=16.04
+        - ARCH=ppc64el INCLUDE=iproute2,wget QEMU_ARCH=ppc64 SUITE=xenial UNAME_ARCH=ppc64el VERSION=16.04
+
+        - ARCH=amd64 INCLUDE=iproute2,wget QEMU_ARCH=x86_64 SUITE=yakkety UNAME_ARCH=x86_64 VERSION=16.10
+        - ARCH=arm64 INCLUDE=iproute2,wget QEMU_ARCH=aarch64 SUITE=yakkety UNAME_ARCH=arm64 VERSION=16.10
+        - ARCH=armhf INCLUDE=iproute2,wget QEMU_ARCH=arm SUITE=yakkety UNAME_ARCH=armv7l VERSION=16.10
+        - ARCH=i386 INCLUDE=iproute2,wget QEMU_ARCH=i386 SUITE=yakkety UNAME_ARCH=i386 VERSION=16.10
+        - ARCH=powerpc INCLUDE=iproute2,wget QEMU_ARCH=ppc SUITE=yakkety UNAME_ARCH=powerpc VERSION=16.10
+        - ARCH=ppc64el INCLUDE=iproute2,wget QEMU_ARCH=ppc64 SUITE=yakkety UNAME_ARCH=ppc64el VERSION=16.10
+
+        - ARCH=amd64 INCLUDE=iproute2,wget QEMU_ARCH=x86_64 SUITE=zesty UNAME_ARCH=x86_64 VERSION=17.04
+        - ARCH=arm64 INCLUDE=iproute2,wget QEMU_ARCH=aarch64 SUITE=zesty UNAME_ARCH=arm64 VERSION=17.04
+        - ARCH=armhf INCLUDE=iproute2,wget QEMU_ARCH=arm SUITE=zesty UNAME_ARCH=armv7l VERSION=17.04
+        - ARCH=i386 INCLUDE=iproute2,wget QEMU_ARCH=i386 SUITE=zesty UNAME_ARCH=i386 VERSION=17.04
+        - ARCH=ppc64el INCLUDE=iproute2,wget QEMU_ARCH=ppc64 SUITE=zesty UNAME_ARCH=ppc64el VERSION=17.04
+
+before_script:
+    - sudo apt-get -y install debootstrap
+    - wget -N http://security.ubuntu.com/ubuntu/pool/universe/q/qemu/qemu-user-static_2.8+dfsg-3ubuntu2_amd64.deb
```

Comment:
Yes, but I have problem with qemu-user-static because travis use older version of ubuntu therefore older version the qemu-user-static which doesn't support ppc64el architecture.